addons/mail/static/src/models/message/message.js

```diff
@@ -482,6 +484,14 @@ function factory(dependencies) {
             );
         }
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+         _computeIsCurrentPartnerMentioned() {
+            return this.messaging.currentPartner && this.recipients.map(r => r.id).includes(this.messaging.currentPartner.id);
```

Comment:
no need to compare ids, directly using includes on the field should work

Also note that if `this.messaging.currentPartner` is not set, returning `undefined` is not the same as returning `false` (but is the check still needed if not using `.id`)?